src/algorithm/joint-configuration.hpp

```diff
@@ -610,6 +610,39 @@ namespace pinocchio
     normalize<LieGroupMap,Scalar,Options,JointCollectionTpl,ConfigVectorType>(model,PINOCCHIO_EIGEN_CONST_CAST(ConfigVectorType,qout));
   }
 
+  /**
+   *
+   * @brief         Check whether a configuration vector is normalized within the given precision provided by prec.
+   *
+   * @param[in]     model      Model of the kinematic tree.
+   * @param[in]     q          Configuration to check (size model.nq).
+   * @param[in]     prec       Precision.
+   *
+   * @return     Whether the configuration is normalized or not, within the given precision.
+   */
+  template<typename LieGroup_t, typename Scalar, int Options, template<typename,int> class JointCollectionTpl, typename ConfigVectorType>
+  inline bool isNormalized(const ModelTpl<Scalar,Options,JointCollectionTpl> & model,
+                           const Eigen::MatrixBase<ConfigVectorType> & q,
+                           const Scalar prec = Eigen::NumTraits<Scalar>::dummy_precision());
```

Comment:
Please use ref for prec

Reply:
May I know why? I see this is done in `isSameConfiguration` for instance, but I actually did my best to avoid it. Pass-by-const-reference is useless for scalar types and may be counter-productive : https://stackoverflow.com/a/270435/12175820

Reply:
Not in the case of Scalar for Autodiff and so on. In general, we follow the same pattern as Eigen. For isSameConfiguration, it is maybe a mistake.

Reply:
No, I mean `isSameConfiguration` _is_ using ref, but I thought _that_ do be a mistake and I did differently. Ok, I'll change it

Reply:
@gabrielebndn When do you plan to handle the comments?

Reply:
I think I can do it today, maximum by the end of the week